SampleVIPSwift/SampleVIPSwift/SampleVIPSwift/Scenes/FirstScene/FirstSceneFactorable.swift

```diff
@@ -20,14 +23,9 @@ extension AppInjector {
     }
 }
 
-extension AppInjector.FirstSceneInjector: FirstSceneFactorable  {}
+extension AppInjector.FirstSceneInjector: FirstSceneFactorable {}
 
-protocol FirstSceneFactorable: Factorable, ServicesFactorable {
-
-    func makeInteractor(viewController: FirstSceneDisplayLogic?, dataSource: FirstSceneModel.DataSource) -> FirstSceneInteractable
-    func makePresenter(_ viewController: FirstSceneDisplayLogic?) -> FirstScenePresentationLogic
-    func makeRouter(viewController: UIViewController?) -> FirstSceneRouting
-}
```

Comment:
Removing this, will affect the unitTests part,
Custom implementation  for testing (like making my own Interactor for this ViewController) won't be called, instead the default implementation is called